<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>790</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1268</width>
    <height>790</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1268</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Config">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Save"/>
    <addaction name="action_Load"/>
    <addaction name="separator"/>
    <addaction name="action_Clear"/>
    <addaction name="separator"/>
    <addaction name="actionE_Xit"/>
   </widget>
   <widget class="QMenu" name="menu_PyRQ">
    <property name="title">
     <string>PyRQ</string>
    </property>
    <addaction name="action_add"/>
    <addaction name="action_remove"/>
    <addaction name="separator"/>
    <addaction name="actionSet_Host"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Queue">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Queue</string>
    </property>
    <addaction name="action_Create"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Get"/>
    <addaction name="action_Put"/>
    <addaction name="action_QSize"/>
    <addaction name="action_MaxQSize"/>
   </widget>
   <addaction name="menu_Config"/>
   <addaction name="menu_PyRQ"/>
   <addaction name="menu_Queue"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save settings</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="text">
    <string>&amp;Load settings</string>
   </property>
  </action>
  <action name="action_add">
   <property name="text">
    <string>&amp;Add</string>
   </property>
  </action>
  <action name="action_remove">
   <property name="text">
    <string>&amp;Remove</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionE_Xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionSet_Host">
   <property name="text">
    <string>Set &amp;Host</string>
   </property>
  </action>
  <action name="action_Create">
   <property name="text">
    <string>&amp;Create</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>C&amp;lose</string>
   </property>
  </action>
  <action name="action_Get">
   <property name="text">
    <string>&amp;Get</string>
   </property>
  </action>
  <action name="action_Put">
   <property name="text">
    <string>&amp;Put</string>
   </property>
  </action>
  <action name="action_QSize">
   <property name="text">
    <string>Q&amp;size</string>
   </property>
  </action>
  <action name="action_MaxQSize">
   <property name="text">
    <string>&amp;MaxQSize</string>
   </property>
  </action>
  <action name="action_Clear">
   <property name="text">
    <string>Clear settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
